colossalai/shardformer/README.md

```diff
@@ -390,3 +390,32 @@ To validate that training the model using shardformers does not impact its conve
 | 0.81855  | 0.87124 | 0.10357 |     1      |    False    |
 
 Overall, the results demonstrate that using shardformers during model training does not affect the convergence.
+
+### performance
+
+We also conducted [benchmark tests](./examples/performance_benchmark.py) to evaluate the performance improvement of Shardformer. We compared the training time between the original model and the shard model.
+
+In the case of using 2 GPUs, the training times are as follows:
+| N_CTX |   org_model    |  shard_model   |
```

Comment:
You need to explain the meaning of N_CTX and what kind of configurations are turned on for benchmarking.